src/server/scripts/Spells/spell_item.cpp

```diff
@@ -3456,9 +3470,16 @@ class spell_item_trinket_stack : public SpellScriptLoader
                 caster->CastSpell(target, _triggerSpell, aurEff);
         }
 
+        void OnRemove(AuraEffect const*, AuraEffectHandleModes)
+        {
+        	if (GetTarget())
+       			GetTarget()->RemoveAurasDueToSpell(_stackSpell);
```

Comment:
seems like tabs or wrong spaces

Reply:
Yup, @DanVS needs to configure editor to auto convert tabs to 4 spaces.